cmd/goProbe/flags/root.go

```diff
@@ -19,9 +21,11 @@ var CmdLine = &Flags{}
 // Read reads in the command line parameters
 func Read() error {
 	flag.StringVar(&CmdLine.Config, "config", "", "path to goProbe's configuration file (required)")
-	flag.BoolVar(&CmdLine.DocGen, "docgen", false, "generate API documentation and exit. A configuration file has to be provided with -config")
 	flag.BoolVar(&CmdLine.Version, "version", false, "print goProbe's version and exit")
 
+	flag.BoolVar(&CmdLine.Profiling, "profiling", false, "enable profiling")
```

Comment:
Interestingly, I had it like that at first. Happy to change it back to the `profiling-output-dir` setup. And of course create the directory if it doesn't exists.

As for the include of `https/pprof`, I didn't find any references to it inside `runtime/pprof`. On that note though: I want to also spawn an http/pprof server in case the API is enabled and supply that handler for visualizing flame graphs.

I will tackle that in the issue that refactors the API (move to gin-gonic, standardized paths that are more RESTy, etc.). Issue to follow.